src/lib/datepicker/datepicker.ts

```diff
@@ -167,6 +167,9 @@ export class MatDatepicker<D> implements OnDestroy {
    */
   @Output() selectedChanged = new EventEmitter<D>();
 
+  /** Classes to be passed to the date picker panel. Supports the same syntax as `ngClass`. */
+  @Input() panelClass: string|string[]|Set<string>|{[key: string]: any};
```

Comment:
I think `Set<string>|{[key: string]: any}` is a bit overkill. I'd rather stick with `string | string[]` for the time being.